dm/dm/ctl/master/query_validation.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+	"os"
+
+	"github.com/pingcap/errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap/tiflow/dm/dm/ctl/common"
+	"github.com/pingcap/tiflow/dm/dm/pb"
+)
+
+const (
+	ValidationStageRunning = "running"
+	ValidationStageStopped = "stopped"
+	ValidationAllErr       = "all"
+	ValidationIgnoredErr   = "ignored"
+)
+
+func NewQueryValidationErrorCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "show-errors [--error] <task-name>",
+		Short: "show error of the validation task",
+		RunE:  queryValidationError,
+	}
+	cmd.Flags().String("error", "all", "filtering type of error: all (default) or ignored")
```

Comment:
Are non-ignored errors are a subset of all errors? 
And what kinds of errors should be reported if the `error` flag is not set? I thought `all` is the default value since we should report all kinds of errors (including `ignored` and `non-ignored`, etc.) if users don't set it. 
According to the spec, it seems only these two values are accepted. Is `non-ignored` acceptable here?